lib/index.js

```diff
@@ -284,44 +256,83 @@ export class Baker extends EventEmitter {
 
   /**
    * Prepares a function to process crosswalk image paths.
-   * @param {string} crosswalkPath - The base path for crosswalk images.
    * @returns {Function} A function that maps over data entries to update image paths.
    */
-  prepareCrosswalk(/** @type {string} */ crosswalkPath) {
+  prepareCrosswalk() {
     /**
      * Processes data entries to set appropriate paths for images.
      * @param {Object.<string, string>[]} data - Array of objects, each containing image references.
      * @returns {Object.<string, string>[]} The data with updated image paths.
      */
-    return function setCrosswalkPath(
+    return function setCrosswalkDefaultPaths(
       /** @type {Object.<string, string>[]} */ data
```

Comment:
```suggestion
      /** @type {Object.<string, string>} */ data
```

Reply:
It's really an object of objects. I'll update